app/blog/_posts/2015-12-28-our-year-in-review-2015.md

```diff
@@ -0,0 +1,106 @@
+
+{% include post-image.html alt="Hjörtur and his son meditating" src="/images/blog/year-in-review/marco-talk_16_9.jpg" margin="both" ratio="16_9" %}
+{% include post-image.html alt="Hjörtur and his son meditating" src="/images/blog/year-in-review/wp-meetup-hjortur_16_9.jpg" margin="both" ratio="16_9" %}
+{% include post-image.html alt="Hjörtur and his son meditating" src="/images/blog/year-in-review/david_hyper_talk.jpg" margin="both" ratio="16_9" %}
+
+
+## In the papers
+
+Breaking news, or not really, but we had our first newspaper appearance in a newspaper in Iceland. Its called [Fréttabladid](http://www.visir.is/section/FRETTABLADID) and is the biggest traditional paper in Iceland. That was something new.
+
+{% include post-image.html alt="Icelandic newspaper" src="/images/blog/year-in-review/icelandic-newspaper.jpg" margin="both" ratio="1_1" %}
+
+
+## Wave of (cool) client work
+
+The year ended with lots of cool projects. We did the [Oyyo](http://oyyo.se/) webshop using [Shopify](https://shopify.com) with our friends at [Studio Källbom](http://www.studiokallbom.se/). We built a new website for [SPG](http://spg.se/) with our friends at [BVD](http://bvd.se/). And we worked on the [Facebook TechPrep](https://techprep.fb.com/) website and [Google Santa Tracker 2015](https://santatracker.google.com/#village) with our friends at [Upperquad](http://upperquad.com/), and many others.
+
+{% include post-image.html alt="Santa Tracker 2015" src="/images/blog/year-in-review/santa-screenshot_16_9.png" margin="both" ratio="16_9" %}
+
+
+## Julbord in Sandhamn
+
+After all the hard work, talks and traveling, the 14islanders decided to reap their harvest in [Sandhamn](https://en.wikipedia.org/wiki/Sandhamn), an island in the beautiful archipelago located in Stockholm. We filled our stomachs with traditional [Swedish christmas-food](https://sweden.se/culture-traditions/christmas/) and sung Swedish drinking songs as we toasted the traditional schnaps.
+
+{% include post-image.html alt="Julbord in Sandhamn" src="/images/blog/year-in-review/julbord_16_9.jpg" margin="both" ratio="16_9" %}
+
+## Hej då, 15. Hejsan, 16.
+
+We want to thank all our friends and the people we been working with for an amazing year. We really couldn’t do any of this without you.  Marry Christmas everyone, lets make next year count!
```

Comment:
Since we later then we planned with the post, maybe we should say "Happy new year everyone", instead of saying "Marry Christmas everyone". Its also "Merry" instead of "Marry" (My mistake).